cms/djangoapps/contentstore/views/course.py

```diff
@@ -728,15 +729,16 @@ def course_info_update_handler(request, course_key_string, provided_id=None):
 
 
 @require_http_methods("POST")
-def send_test_enrollment_email(request):
+def send_test_enrollment_email(request, course_key_string):
     """
     Handles ajax request for sending test enrollment emails to the instructor
     """
+    course_key = CourseKey.from_string(course_key_string)
     user = request.user
     from_address = microsite.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
     subject = request.POST.get('subject')
     subject = ''.join(subject.splitlines())
-    message = request.POST.get('message')
+    message = substitute_keywords_with_data(request.POST.get('message'), user.id, course_key)
```

Comment:
I tend to prefer making this kind of change like the following:
```python
message = request.POST.get('message')
message = substitute_keywords_with_data(message, user.id, course_key)
```

- The diff is cleaner, as it's just an added line, not both an added and
  removed line.
- It's easier to debug, as you can step through lines of code and see
  how the variable changes from call to call.
- It's easier to read, as nested function calls often lead to longer
  lines which are more difficult to mentally parse with levels of nested
  parens.